readthedocs/telemetry/collectors.py

```diff
@@ -92,8 +92,27 @@ def collect(self):
             },
         }
         data["doctool"] = self._get_doctool()
+
+        # NOTE: Check if there are files at `_build/html` (the old output directory)
+        # and log these projects so we can communicate with their maintainers
+        #
+        # This temporal and should be removed in the future once we have
+        # decided what to do with this collected data.
+        self._check_using_old_output_directory()
+
         return data
 
+    def _check_using_old_output_directory(self):
+        code, stdout, stderr = self.run(
+            "ls",
+            "_build/html",
+            cwd=self.project.checkout_path(self.version.slug),
+        )
+        if code == 0:
+            log.warning(
+                "Directory '_build/html' exists. This may lead to unexpected behavior."
```

Comment:
We should raise something user facing as well. I'm still a huge +1 on better error/warning patterns, but would it make sense to set the `Build.error` here? If there is something in this path, that almost certainly mean the build will be missing content if we don't fail the build explicitly.